### ISSUE TITLE
About the number of epoches

### ISSUE THREAD

**Issue by LiuZhenYu-BJFU** [JunlinHan/DCLGAN#9]
Hello!
Thanks for open source your code.
I was training the summer2winter_yosemite dataset, but when I trained 50 epochs, I tested FID=115.7,  when I trained 100 epochs, I found that the FID was greatly improved, and I checked the index and found that it was indeed trained later When fakeB becomes very inexplicable.
the following pictures are the effects of 50epoch and 100epoch, it is obvious that 100epoch becomes very strange
![image](https://user-images.githubusercontent.com/74093654/162729788-4b6ff052-ea3e-4e0a-9bab-5ba7b22f3fd1.png)
![image](https://user-images.githubusercontent.com/74093654/162729763-99dbe2c9-4057-4ace-b02b-f69fa4214001.png)

And there seems to be something wrong with the names displayed on visdom (please see the idt_B and real_B). Aren't they supposed to be very similar?
![image](https://user-images.githubusercontent.com/74093654/162730212-6229c591-8e30-4a89-a791-d0a91ab3056b.png)

I would appreciate it if you could reply!
Thank you!

**Comment by JunlinHan** [comment]
Hello Liuzhen,
Thanks for your attention in our work!

1: 50ep/100ep
If possible, would you mind sharing more details (traiing setting) ? Especially, learning rate, batch_size, and total training epoch in your setting.

2: idt_A/idt_B
Oh that's not an error. Idt_b is passing real_A to a Generator with translation direction A->B, so idt_b should be similar to real_A.

**Comment by ZhenyuLiu-BJFU** [comment]
Hello Junlin,
Thank you for your reply!

1. I have not changed your setup parameters(but maybe when I train the first 50 epoch, I change the batch_size=2. But when I train the second 50 epoch, I change it back to 1). And when I train the second time I use the --epoch_count 50 --continue_train. 
I don't know if this will work because I follow CycleGAN‘s operation.
2. Thanks for the explanation, I guess I should have gotten confused with the other paper, I'll go back and read the details of the article.

**Comment by JunlinHan** [comment]
> 
Hello again,

1: Oh most setting in this repo follows CycleGAN, so it's ok to follow CycleGAN. Batch_size = 1 would be better

Regarding FID: Usually, the highest FID occurs during the middle of training. But higher FID does not always guarantee better results. It's better to train for ~1000000 iterations.

2: The naming actually follows CycleGAN, which might be confusing.
